pkg/lockfile/dpkg-status.go

```diff
@@ -0,0 +1,105 @@
+package lockfile
+
+import (
+	"bufio"
+	"fmt"
+	"os"
+	"strings"
+)
+
+const DebianEcosystem Ecosystem = "Debian"
+
+func groupDpkgPackageLines(scanner *bufio.Scanner) [][]string {
```

Comment:
noting that while this is exactly the same as what's in `parse-apk-installed` because of how Go compiles I don't think it's worth making into a utility function until we see it being repeated another time (the yarn parser has one that is very similar and you could deduplicate, but it'd require some extra complexity with passing in functions that I also don't think is worth it)